router/internal/codemode/sandbox/sandbox_preamble.js

```
@@ -0,0 +1,75 @@
+  if (v && typeof v === "object") {
+    const out = {};
+    for (const k in v) {
+      const c = compact(v[k]);
+      if (c != null) out[k] = c;
+    }
+    return out;
+  }
+  return v;
+};
+
+delete globalThis.eval;
```

Biome (2.4.14)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)